Feat/currency string adjustment

jdsmith022 opened the pull request:
Size of font and logo are relative to the length of the compensation now. Did a little refactor of the code->removed duplicate code, cart and checkout button go to same function minimal and default button functions successfully as compensations string is being used rather than quantity of items, and made consistency changes 

Tested in all themes in WC-test.co2ok.eco

**Sample Result:**

<img width="660" alt="Screen Shot 2020-10-20 at 1 02 39 PM" src="https://user-images.githubusercontent.com/45078521/96577954-bc82fd80-12d4-11eb-9622-1aa2819217be.png">
<img width="676" alt="Screen Shot 2020-10-20 at 1 02 45 PM" src="https://user-images.githubusercontent.com/45078521/96577961-be4cc100-12d4-11eb-9486-dd692739c629.png">

<img width="660" alt="Screen Shot 2020-10-20 at 12 57 58 PM" src="https://user-images.githubusercontent.com/45078521/96578073-e63c2480-12d4-11eb-84ba-51acc317093a.png">
<img width="538" alt="Screen Shot 2020-10-20 at 12 58 04 PM" src="https://user-images.githubusercontent.com/45078521/96578076-e89e7e80-12d4-11eb-9ef2-1f221c6de915.png">
<img width="677" alt="Screen Shot 2020-10-20 at 12 57 50 PM" src="https://user-images.githubusercontent.com/45078521/96578077-e9371500-12d4-11eb-9723-517a4f32c6a8.png">
<img width="682" alt="Screen Shot 2020-10-20 at 12 57 43 PM" src="https://user-images.githubusercontent.com/45078521/96578081-e9cfab80-12d4-11eb-8c3a-33cf88a1277b.png">
<img width="638" alt="Screen Shot 2020-10-20 at 12 58 41 PM" src="https://user-images.githubusercontent.com/45078521/96578083-e9cfab80-12d4-11eb-911d-dfeea161554e.png">
<img width="663" alt="Screen Shot 2020-10-20 at 12 58 47 PM" src="https://user-images.githubusercontent.com/45078521/96578085-ea684200-12d4-11eb-8a84-9ec6d106a337.png">




**Discrepancies found for two themes:** 

Woodmart: 
minimal: 
<img width="436" alt="Screen Shot 2020-10-20 at 12 25 13 PM" src="https://user-images.githubusercontent.com/45078521/96574986-775ccc80-12d0-11eb-9eb1-411b6a5a2544.png">
default:
<img width="261" alt="Screen Shot 2020-10-20 at 12 30 08 PM" src="https://user-images.githubusercontent.com/45078521/96575012-7e83da80-12d0-11eb-8f47-bdd487ec19a1.png">


27:
default:
<img width="305" alt="Screen Shot 2020-10-20 at 11 31 25 AM" src="https://user-images.githubusercontent.com/45078521/96575023-82176180-12d0-11eb-861f-e952a4cc4c6c.png">

Mil0dV commented on css/co2ok.css:
```diff
@@ -343,9 +343,9 @@ img#co2ok_logo_minimal {
       padding-left: 19px;
       left: -10px;
-      margin-top: 9px !important;
+      margin-top: 9px;
```
Usually, the `!important`'s are there for a reason. Sometimes finding the corresponding commit with `git blame` tells us why we had to include the important. 

In general I would not remove them unless necessary.

Mil0dV replied:
This works in my dev tools:
```
var element = document.querySelector('.compensation_amount_default');
element.setAttribute( 'style', 'font-size: 23px !important' );
```

But if that still gives issues, let's leave it - I think our previous code already reset the !importants :)

jdsmith022 replied:
Yes, I tried this and also `element.style.cssText = 'font-size: 22px !important';` and it wouldn't overwrite the css

TheApeMachine replied:
https://css-tricks.com/fitting-text-to-a-container/ why not just do it in CSS and avoid the complexity of a JS solution...

Mil0dV reviewed (changes requested):
Was it necessary to shrink 'Make' and the logo? I'd prefer to only shrink the amount unless that looks weird/leads to side effects. 

Ideally we could then clean up default- and minimalButton some more (e.g. the else shouldn't be needed since those values are in the css, right?)

Mil0dV commented on js/co2ok-plugin.js:
```diff
@@ -1,81 +1,108 @@
+  var comp_amount_minimal = document.querySelector('.comp_amount_label_minimal');
+
+  var cad_length_minimal = cad_minimal.innerHTML.length;
+  var relative_font_size;
```
in JS you don't need to declare vars first :)

Mil0dV commented on js/co2ok-plugin.js:
```diff
@@ -151,171 +176,64 @@ var Co2ok_JS = function ()
+                    // var make_minimal = document.querySelector('.make_co2ok_minimal');
 
-                    if(document.querySelector('.qty') != null){
+                    if(document.querySelector('.qty') != null) {
```
this can go, right? (and some code below too)

jdsmith022 commented:
removed more unnecessary code left over from .qty and .quantity-product. removed else statements from minimalButton() and defaultButton() and added few style changes into css instead.

Ghostyfied commented on js/co2ok-plugin.js:
```diff
@@ -1,702 +1,542 @@
+
+                      if(co2ok_temp_global.id == 'default_co2ok_temp') {
+
+                        defaultButton();
```
Does this work without scope??

Mil0dV replied:
The WC plugin JS is a mess - this function is defined at the top level - global scope.

jdsmith022 replied:
^^ yep

Ghostyfied commented on js/co2ok-plugin.js:
```diff
@@ -1,702 +1,542 @@
+
+                      } else {
+
+                        minimumButton();
```
same ^^

Mil0dV reviewed (changes requested):
Only fix if it's very quick
